CMSgov/mint-app
# [MINT-3043] Front End work to create Documents Card

# MINT-3043

## Description

* Created `DocumentsCard` component and its corresponding files
* Add copy to `collaborationArea.ts`
* Add `fileType` to modelPlan query
* Updated `documents.spec.js` cypress test
* Moved shared styles from `ModelPlanCard/index.scss` and `DocmentsCard/index.scss` into the `index.scss` in parent folder
* Removed `DocumentBanner` from `Tasklist`


## How to test this change

1.  Navigate to model collaboration area and verify that Documents card is there
2. Click button to add document
3. Go through the steps to add a link
4. Return to the model collaboration area and verify that it now says "1 link added"
5. Go through the steps to add another link and verify that it now says "2 links added"
6. Go through the steps to add a PDF and verify that it now says "1 uploaded"
7. Verify that `View all` button shows up only when there is something added/uploaded
8. Verify that breadcrumbs are updated
NOTE: If accessing "Add Documents" directly from the collaboration area, hide the Documents breadcrumb

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
